### ISSUE TITLE
Why vue version is 2.2.x in nuxt/0.10.7 ?

### ISSUE THREAD

**Issue by jsnick** [nuxt/nuxt.js#705]
I checked vue "~2.3.x" was built in nuxt/v0.10.7

But If I install 0.10.7, nuxt has vue "~2.2.6".
This is `node_modules/nuxt/package.json`.
```
{
  "name": "nuxt",
  "version": "0.10.7",
  "description": "A minimalistic framework for server-rendered Vue.js applications (inspired by Next.js)",
  "contributors": [
    {
      "name": "Sebastien Chopin"
    },
    {
      "name": "Alexandre Chopin"
    }
  ],
  ...
  ...
  "dependencies": {
    ...
    ...
    "vue": "~2.2.6",
    "vue-loader": "^11.3.4",
    "vue-meta": "^0.5.6",
    "vue-router": "^2.4.0",
    "vue-server-renderer": "~2.2.6",
    "vue-ssr-html-stream": "^2.2.0",
    "vue-ssr-webpack-plugin": "^3.0.0",
    "vue-template-compiler": "~2.2.6",
    "vuex": "^2.3.0",
    "webpack": "^2.4.1",
    "webpack-bundle-analyzer": "^2.4.0",
    "webpack-dev-middleware": "^1.10.1",
    "webpack-hot-middleware": "^2.18.0"
  },
  ...
}
```

I tried to install both yarn (no cached) and npm but vue version is still 2.2.6.
You can see this if just `npm install nuxt`

**Comment by pi0** [comment]
@jsnick Vue 2.3.x is not available in latest `0.10.7` release. It may introduce some small breaking changes as of SSR improvements. It is coming soon :)

**Comment by jsnick** [comment]
Oh I see. Thanks @pi0
